processor/src/main/java/org/mapstruct/ap/internal/model/source/ValueMapping.java

```diff
@@ -0,0 +1,165 @@
+/**
+ *  Copyright 2012-2016 Gunnar Morling (http://www.gunnarmorling.de/)
+ *  and/or other contributors as indicated by the @authors tag. See the
+ *  copyright.txt file in the distribution for a full listing of all
+ *  contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package org.mapstruct.ap.internal.model.source;
+
+import java.util.List;
+import javax.lang.model.element.AnnotationMirror;
+import javax.lang.model.element.AnnotationValue;
+import javax.lang.model.element.ExecutableElement;
+import org.mapstruct.ap.internal.prism.ValueMappingPrism;
+import org.mapstruct.ap.internal.prism.ValueMappingsPrism;
+import org.mapstruct.ap.internal.util.FormattingMessager;
+import org.mapstruct.ap.internal.util.Message;
+
+/**
+ * Represents the mapping between one value constant and another.
+ *
+ * @author Sjaak Derksen
+ */
+public class ValueMapping {
+
+    public static final String ANY = "<ANY>";
```

Comment:
Hmm, or additionally put the constants in a class next to our existing enum-prisms (e.g. `MappingConstantsPrism`), to emphasise that it reflects the content of that other class in the API.